app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+# app/controllers/users_controller.rb
+class UsersController < ApplicationController
+  before_action :set_user, only: [:show]
+
+  def index
+    # Retrieve a list of users (e.g., User.all) and assign it to an instance variable
+    @users = User.all
+  end
+
+  def show
+    # The "set_user" method (defined below) fetches the user by ID, and we assign it to an instance variable
+    # This action will display the details of a specific user
+  end
```

Comment:
[REQUIRED] Kindly remove unused actions an unnecessary comments. This will make your code more professional.